docs/specifications/tests/DNSSEC-TP/dnssec16.md

```diff
@@ -0,0 +1,224 @@
+# DNSSEC16: Validate CDS
+
+## Test case identifier
+**DNSSEC16**
+
+## Objective
+
+CDS and CDNSKEY record types are defined in [RFC 7344] and [RFC 8078].
```

Comment:
Should we really be speaking of CDNSKEY in the objective section of DNSSEC16? It makes sense to mention it in the scope section. But having it in the Objective section makes it look like this test case also concerns itself with CDNSKEY.

And conversely in DNSSEC17.

Reply:
I think it is. They are very connected. It says "The objective of this test
case is to verify that the CDS RRset is valid" and "For tests of the
CDNSKEY, see test case [DNSSEC17]".